src/cbmc_starter_kit/template-for-repository/proofs/Makefile.common

```diff
@@ -682,75 +685,76 @@ else
 	  --description "$(PROOF_UID): setting static variables to nondet"
 endif
 
-# Omit unused functions (sharpens coverage calculations)
+# Omit initialization of unused global variables (reduces problem size)
```

Comment:
we should move this after contract instrumentation, same step as "omit unused functions"

Reply:
Implemented as suggested.